.golangci.yml

```diff
@@ -3,14 +3,27 @@ linters-settings:
     disable:
       - shadow # default value recommended by golangci
       - composites
+  
+linters:
+  enable:
+    - gosec
 
 run:
   build-tags:
     - integration
+  timeout: 5m
 
 issues:
   # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
   max-issues-per-linter: 0
 
   # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
   max-same-issues: 0
+
+  exclude-rules:
+    - text: "G404:" # Ignore weak random number generator lint. We do not need strong randomness here.
+      linters:
+        - gosec
+    - text: "G601:" # Ignore Implicit memory aliasing in for loop
```

Comment:
Did you check that loops that break this rule aren't buggy? They feel suspicious to me.

Reply:
I checked some of them but not all. I seemed to me that unintended bugs would be really noticeable in this case.

Reply:
I will do local excludes in every case and review the loops